pallets/task/src/tests.rs

```diff
@@ -136,178 +170,174 @@ fn increase_task_count_when_creating_task(){
 #[test]
 fn increase_task_count_when_creating_two_tasks(){
 	new_test_ext().execute_with( || {
-
 		// Profile is necessary for task creation
-		assert_ok!(Profile::create_profile(Origin::signed(1), username(), interests(), HOURS, Some(additional_info())));
+		assert_ok!(Profile::create_profile(Origin::signed(*ALICE), username(), interests(), HOURS, Some(additional_info())));
 
 		// Ensure new task can be created.
-		assert_ok!(Task::create_task(Origin::signed(1), title(), spec(), BUDGET, get_deadline(), attachments(), keywords()));
-		assert_ok!(Task::create_task(Origin::signed(1), title(), spec2(), BUDGET2, get_deadline(), attachments(), keywords()));
+		assert_ok!(Task::create_task(Origin::signed(*ALICE), title(), spec(), BUDGET, get_deadline(), attachments(), keywords(), None));
+		assert_ok!(Task::create_task(Origin::signed(*ALICE), title(), spec2(), BUDGET2, get_deadline(), attachments(), keywords(), None));
 
 		// Assert that count is incremented to 2 after task creation
 		assert_eq!(Task::task_count(), 2);
 	});
 }
 
 #[test]
-fn cant_own_more_tax_than_max_tasks(){
```

Comment:
Tax :D

Reply:
Should I change it to 'cant_owe_more_tax'? :p